routers/api/v1/org/team.go

```diff
@@ -138,9 +142,45 @@ func GetTeam(ctx *context.APIContext) {
 	//   "200":
 	//     "$ref": "#/responses/Team"
 
+	if err := ctx.Org.Team.GetUnits(); err != nil {
+		ctx.Error(http.StatusInternalServerError, "team.GetUnits", err)
+		return
+	}
+
 	ctx.JSON(http.StatusOK, convert.ToTeam(ctx.Org.Team))
```

Comment:
As for the CI errors, it seems to be happening here that `Authorize` is not being loaded from database into `AccessMode`.

Reply:
I have reverted a change what you suggested. I think that's the reason.